pages.pt_BR/common/mpv.md

```diff
@@ -0,0 +1,36 @@
+# mpv
+
+> Um tocador de vídeo/audio baseado no MPlayer.
+> Mais informações: <https://mpv.io>.
+
+- Tocar um vídeo ou áudio de uma URL ou arquivo:
+
+`mpv {{https://www.youtube.com/watch?v=dQw4w9WgXcQ|caminho/para/arquivo}}'`
+
+- Avançar/retroceder 5 segundos:
+
+`ESQUERDA <ou> DIREITA`
```

Comment:
@sebastiaanspeck What do you suggest doing in this case?

The command contains a keyboard shortcut which depending on the keyboard layout will differ in other languages, should we go with the English version to fix the errors? (Anything works for me)